### ISSUE TITLE
Building on arm64 macos

### ISSUE THREAD

**Issue by dyfer** [flucoma/flucoma-sc#69]
Hello FluCoMa devs,

I just wanted to share an experience building flucoma-sc on arm64 macOS 12.2 / Xcode 13 (Apple M1 hardware).

TL;DR it works!

There's one issue I encountered: flucoma-core / eigen had an [issue](https://github.com/flucoma/flucoma-core/issues/69) building with Xcode 13, so I commented out one line as suggested in https://github.com/flucoma/flucoma-core/issues/69#issuecomment-988029893

Moreover, I had to explicitly specify the architecture: 
```sh
cmake -DSC_PATH=/paht/to//supercollider -DCMAKE_OSX_ARCHITECTURES=arm64 ..
```
I might be wrong, but I don't think the architecture [needs to be hardcoded in CMakeLists.txt](https://github.com/flucoma/flucoma-sc/blob/main/CMakeLists.txt#L25), I believe it should be picked up automatically.

It seems to work well so far.

Thanks so much for a great library! I hope you find this report useful.

**Comment by woutersnoei** [comment]
Hi dyfer, not as a dev but as an interested person; would you be ok to share the binaries you built (with me and/or the rest of the world)? Cheers & thanks!

**Comment by dyfer** [comment]
Here it is. 
This was compiled in February 2022, can't remember if it was done from the main or the dev branch...
No warranty whatsoever :)
[FluidCorpusManipulation.zip](https://github.com/flucoma/flucoma-sc/files/9736502/FluidCorpusManipulation.zip)
